Explorer/Assets/Plugins/DCLCrypto/Authenticator.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+
+namespace DCLCrypto
+{
+    public static class Authenticator
+    {
+        public delegate string SignerFunc(string payload);
+
+        public static string GetEphemeralMessage(string ephemeralAddress, DateTime expiration) =>
+            $"Decentraland Login\nEphemeral address: {ephemeralAddress}\nExpiration: {expiration:s}";
+
+        public static AuthIdentity CreateRandomInsecureAuthIdentity()
+        {
+            var identity = Identity.CreateRandom();
+            var ephemeralIdentity = Identity.CreateRandom();
+
+            SignerFunc signerFunc = payload => identity.Sign(payload);
```

Comment:
Can we pass an identity and call `Sign` directly without a capture?
In which cases do you see the necessity of having a `SignerFunc` as a delegate?

Reply:
The Identity represents your wallet. In this case, it is Random. But if one day we want to implement a real user signature, like using Wallet Connect, we need to delegate that function to use the Wallet Connect Sign.